scripts/evaluate.py

```diff
@@ -1221,7 +1290,7 @@ def get_args():
         "log_filename": args.log,
         "early_stopping": None,
         "measure_option": autotvm.measure_option(
-            builder=autotvm.LocalBuilder(build_func=ndk.create_shared, timeout=15, n_parallel=2),
+            builder=autotvm.LocalBuilder(build_func=ndk.create_shared, timeout=15, n_parallel=1),
```

Comment:
For this task, tuning is not needed (yet), the code corresponding to tuning should not change